simphony/cuds/lattice.py

```diff
@@ -190,66 +155,180 @@ def data(self, value):
         self._data = DataContainer(value)
 
 
-def make_hexagonal_lattice(name, h, size, origin=(0, 0, 0)):
-    """Create and return a 2D hexagonal lattice embedded on the XY-plane
-    in 3D.
+def make_cubic_lattice(name, h, size, org=(0, 0, 0)):
+    """Create and return a 3D cubic lattice.
 
     Parameters
     ----------
     name : str
     h : float
         lattice spacing
-    size : int[2]
+    size : int[3]
         Number of lattice nodes in each axis direction.
-    origin : float[3], default value = (0, 0, 0)
+    org : float[3], default value = (0, 0, 0)
```

Comment:
You're right, it's better not to use abbreviations. It was changed to avoid line shifts but that's obviously not a big deal.